mycroft/skills/media/vocab/__init__.py

```diff
@@ -0,0 +1,130 @@
+
+        intent = IntentBuilder('PrevIntent').require('PrevKeyword')
+        self.register_intent(intent, self.handle_prev)
+
+        intent = IntentBuilder('StopIntent').require('StopKeyword')
+        self.register_intent(intent, self.handle_stop)
+
+        intent = IntentBuilder('PauseIntent').require('PauseKeyword')
+        self.register_intent(intent, self.handle_pause)
+
+        intent = IntentBuilder('ResumeIntent').require('ResumeKeyword')
+        self.register_intent(intent, self.handle_resume)
+
+        intent = IntentBuilder('CurrentlyPlayingIntent')\
+            .require('CurrentlyPlayingKeyword')
+        self.register_intent(intent, self.handle_currently_playing)
+
+    def _register_event_handlers(self):
+        """
+           Register event handlers for stopping currently playing media
+           when new media is started and handlers for lowering media volume
+           while mycroft is speaking.
+        """
+        self.emitter.on('mycroft.media.stop', self.handle_stop)
+        self.emitter.on('recognizer_loop:audio_output_start',
+                        self.lower_volume)
+        self.emitter.on('recognizer_loop:audio_output_end',
+                        self.restore_volume)
+
+    def handle_next(self, message):
+        """
+           handle_next() should be implemented by the skill to switch to next
+           song/channel/video in queue.
+        """
+        logger.info('handle_next not implemented')
+
+    def handle_prev(self, message):
+        """
+           handle_prev() should be implemented by the skill to switch to
+           previous song/channel/video in queue
+        """
+        logger.info('handle_prev not implemented')
+
+    def handle_currently_playing(self, message):
+        """
+           handle_currently_playing() should be implemented to tell the user
+           what is currently playing
+        """
+        logger.info('handle_currently_playing not implemented')
+
+    def play(self):
+        """ Stop currently playing media before starting the new. """
+        logger.info('Stopping currently playing media if any')
+        self.emitter.emit(Message('mycroft.media.stop'))
+
+    def handle_pause(self, message):
+        """ handle_pause() should pause currently playing media """
+        logger.info('handle_pause not implemented')
+
+    def handle_resume(self, message):
+        """ handle_resume() should resume paused media """
+        logger.info('handle_resume not implemented')
+
+    def handle_stop(self, message):
+        """
+           handle_stop() should be implemented to stop currently playing media
+        """
+        logger.info('handle_stop not implemented')
+
+    def stop(self):
+        logger.debug('No stop method implemented')
+
+    def lower_volume(self, message):
+        logger.debug('Lower volume not implemented')
+
+    def restore_volume(self, message):
+        logger.debug('Restore volume not implemented')
```

Comment:
For all of these debug lines, change them to something like `"... not implemented in" + self.name` to give developers more useful information.